packages/console/src/pages/Connectors/components/Guide/index.tsx

```diff
@@ -70,6 +81,7 @@ const Guide = ({ connector, onClose }: Props) => {
     const basePayload = {
       config,
       connectorId,
+      id: conditional(isStandard && callbackConnectorId),
```

Comment:
Should check whether the connector is a social connector?

Reply:
updated